src/site/content/en/blog/media-session/index.md

```diff
@@ -0,0 +1,405 @@
+---
+title: Customizing media notifications and playback controls
+subhead: Give users full control over background playback
```

Comment:
```suggestion
subhead: How to integrate with hardware media keys, customize media notifications, and more.
```

This is just my suggestion on a good subhead. It gives you a clear idea of what the article specifically covers.